.eslintignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
Do we want to ignore json?

Reply:
We have massive json files in the `/data` folder
We can include them but the lint error is very very noisy

Reply:
Im ok to ignore then